nmrglue/fileio/bruker.py

```diff
@@ -2091,7 +2092,7 @@ def rm_dig_filter(
 
 # JCAMP-DX functions
 
-def read_jcamp(filename):
+def read_jcamp(filename, encodings=[locale.getpreferredencoding()]):
```

Comment:
I think it would be preferable to use a single encoding here.  If users want to try multiple encoding they can create a loop over `read_jcamp` with a check for a UnicodeDecodeError.